mtk_logger.cc

```diff
@@ -227,6 +227,7 @@ static char* OPT_erase;  /* erase ? command option */
 static char* OPT_erase_only;  /* erase_only ? command option */
 static char* OPT_log_enable;  /* enable ? command option */
 static char* csv_file; /* csv ? command option */
+static char* OPT_block_size; /* block_size ? command option */
```

Comment:
http://www.rigacci.org/wiki/lib/exe/fetch.php/doc/appunti/hardware/gps/mtk_logger_library_user_manual_1.2_tsi.pdf
According to the documentation an MTK data may be requested with any even length.
If the requested length is larger than 0x800, an MTK device returns several output NMEAs with the maximum size 0x800 bytes.

In current implementation of the mtk_logger module the returned chunk that contains only 0xff bytes is used as a marker for the end of the MTK data.
So the reminder of the requested length by 0x800 should be large enough to avoid false recognition for the end of the data.

The sequences of 0xff bytes are most common for the Sector Pattern (the header of the 64K sector) that may contains up to 486 bytes and for the tail of the 64k sector with length that not enough for storing a GPS log record.
But I don't know exactly what is the maximum size of a sequence that can appears.

I think it's reasonable to assume that there's no problem with current request size equals to 1024 and restrict the block_size to be a multiple of 1024.